server/container/guice/guice-common/src/main/java/org/apache/james/modules/server/MailStoreRepositoryModule.java

```diff
@@ -61,14 +59,17 @@ protected void configure() {
 
     @Provides
     @Singleton
-    MailRepositoryStoreConfiguration provideConfiguration(ConfigurationProvider configurationProvider, DefaultItemSupplier defaultItemSupplier) throws ConfigurationException {
+    MailRepositoryStoreConfiguration provideConfiguration(ConfigurationProvider configurationProvider,
+                                                          Provider<MailRepositoryStoreConfiguration.Item> defaultConfigurationItem)
+        throws ConfigurationException {
```

Comment:
I would have put that on the line above, this looks odd when I read it

Reply:
It would make the line very long, I didn't found a good solution for it.